pkg/apis/certmanager/validation/issuer.go

```diff
@@ -19,9 +19,9 @@ package validation
 import (
 	"crypto/x509"
 	"fmt"
-	"strings"
-
 	"github.com/jetstack/cert-manager/pkg/issuer/acme/dns/rfc2136"
+	"net/url"
+	"strings"
```

Comment:
nit: can you keep the separation of stdlib, external and internal packages for imports?

Reply:
Done.